app/src/main/java/com/sdpteam/connectout/event/EventCreatorModel.java

```diff
@@ -0,0 +1,80 @@
+package com.sdpteam.connectout.event;
+
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.ValueEventListener;
+import com.sdpteam.connectout.IdValueListener;
+
+public class EventCreatorModel implements EventDataManager {
+    public final static String DATABASE_EVENT_PATH = "Events";
+    private final DatabaseReference database;
+
+    public EventCreatorModel() {
+        database = FirebaseDatabase.getInstance().getReference();
+    }
+
+    /**
+     * @inheritDoc
+     */
+    @Override
+    public boolean saveValue(Event event) {
+        if (event != null) {
+            database.child(DATABASE_EVENT_PATH).child(event.getEventId()).setValue(event);
+            return true;
+        }
+        return false;
```

Comment:
This conversation has been resolved IRL, it has been agreed that the returned boolean should be preserved as it will be useful for caching & offline mode.